src/box/memtx_tx.h

```diff
@@ -539,6 +539,63 @@ memtx_tx_snapshot_clarify(struct memtx_tx_snapshot_cleaner *cleaner,
 	return memtx_tx_snapshot_clarify_slow(cleaner, tuple);
 }
 
+#if defined(ENABLE_READ_VIEW)
```

Comment:
Let's move the declarations to the EE repo as well and include `memtx_tx_read_view.h` here under `ifdef`.

Also, let's put the `include` directives after all declarations (in the end of the file).

Reply:
Done.